src/mero_conf.erl

```diff
@@ -267,24 +287,26 @@ process_value({servers, {mfa, {Module, Function, Args}}}) ->
 process_value(V) ->
     V.
 
-get_elasticache_cluster_configs({Host, Port, ClusterSpeedFactor}) ->
-    lists:duplicate(ClusterSpeedFactor, get_elasticache_cluster_config(Host, Port));
-get_elasticache_cluster_configs({Host, Port}) ->
-    [get_elasticache_cluster_config(Host, Port)].
+get_elasticache_cluster_configs({Host, Port, ClusterSpeedFactor}, Retries) ->
+    lists:duplicate(ClusterSpeedFactor, get_elasticache_cluster_config(Host, Port, Retries));
+get_elasticache_cluster_configs({Host, Port}, Retries) ->
+    [get_elasticache_cluster_config(Host, Port, Retries)].
 
-get_elasticache_cluster_config(Host, Port) ->
+get_elasticache_cluster_config(Host, Port, Retries) ->
     get_elasticache_cluster_config(Host,
                                    Port,
                                    0,
+                                   Retries,
                                    mero_elasticache:get_cluster_config(Host, Port)).
 
-get_elasticache_cluster_config(_Host, _Port, _Retries, {ok, Entries}) ->
+get_elasticache_cluster_config(_Host, _Port, _Retry, _Retries, {ok, Entries}) ->
```

Comment:
this could use some better naming to distinguish the current retries from the max allowed retries